e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -72,6 +73,21 @@ describe('I18456 Built-in slash commands: common', () => {
             cy.get(`#postMessageText_${postId}`).should('have.text', 'test ¯\\_(ツ)_/¯');
         });
     });
+
+    it('MM-T2345 /me on RHS', () => {
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+        cy.postMessage(MESSAGES.MEDIUM);
+
+        // # Open RHS (reply thread)
+        cy.clickPostCommentIcon();
+
+        // # type /me test
+        cy.get('#reply_textbox').type('/me test');
+        cy.get('#addCommentButton').click();
+
+        // * Properly formatted with lower opacity
+        cy.get('div[id="postListContent"]').contains('p', 'test').should('be.visible');
```

Comment:
Could someone please give me a suggestion on how to test for opacity?

Reply:
The way we're setting up opacity is not via CSS `opacity` but with `color`, so one option is,
```javascript
.and('have.css', 'color', 'rgba(61, 60, 64, 0.6)');
```
(Brittle but should work before we go full into visual regression tests.)

Reply:
ahhh...not ideal but ok for now.